packages/esm-patient-registration-app/src/patient-registration/field/address/address-search.component.tsx

```diff
@@ -66,17 +66,6 @@ const AddressSearchComponent: React.FC<AddressSearchComponentProps> = ({ address
         labelText={t('searchAddress', 'Search address')}
         placeholder={t('searchAddress', 'Search address')}
         ref={searchBox}
-        helperText={
```

Comment:
I'm guessing this triggered some warnings?

Reply:
Yeah, not sure why this got added because the [Search](https://react.carbondesignsystem.com/?path=/docs/components-search--overview) component does not have a helperText prop, and that I couldn't find evidence of this feature working in the UI. Jest also flags a missing prop warning for this in the test console output.